### ISSUE TITLE
Invitation to work on OpenAgar

### ISSUE THREAD

**Issue by Andrews54757** [Barbosik/MultiOgar#410]
@Barbosik , I have seen your very advanced skills and was impressed to the greatest degree. Now, I am creating a new software called OpenAgar. I am inviting you to work on it with us. Everything on OpenAgar is efficient except for collisions. OpenAgar has the potential currently to be 10 times faster than MultiOgar if it wasn't for the collisions slowing it down. We need help from you to perfect it. I am also notifying others of the "Big Three" (Ogar, MultiOgar,OgarUL) for help. If you decline, I will be sad, yet I will yield. Or, you can give us permission to use some of your code. Please join us. I envision a future where a server can hold 1000 players and 500 bots without any lag at all. 

Andrews54757
AJS Development

**Comment by CalvinDKL** [comment]
Come onnn!!! Barbosik join him!

**Comment by MastaCoder** [comment]
@Barbosik is pretty much gone now, the MultiOgar project hasn't be updated for months now.

**Comment by nuclearc** [comment]
@Andrews54757 i think Java, C#, C or C++ will be much better to hold 1k players without any lag

**Comment by Andrews54757** [comment]
yes, but those languages arnt universal. nodejs works on any device

**Comment by PortPixel** [comment]
> yes, but those languages aren't universal. node.js works on any device

Node.js can't run on mobile devices, and you need separate code for the client anyway

**Comment by nuclearc** [comment]
also main target platforms are Windows and Linux, you can compile C++ code for them and it'll run a lot faster than NodeJS

**Comment by G047** [comment]
^ You would have to make the server in such a way that it compiles on all the systems (mac, linux, win) 
doing that on a large project like this is a pain in the ass, also C++ has terrible web socket support.
